jwst/ramp_fitting/tests/test_ramp_fit_cases.py

```diff
@@ -18,6 +19,18 @@
 #  their SCI values (these are mostly for my reference).
 #
 
+test_dq_flags = {
+    "DO_NOT_USE": dqflags.pixel["DO_NOT_USE"],
+    "JUMP_DET": dqflags.pixel["JUMP_DET"],
+    "SATURATED": dqflags.pixel["SATURATED"],
+    "NO_GAIN_VALUE": dqflags.pixel["NO_GAIN_VALUE"],
+    "UNRELIABLE_SLOPE": dqflags.pixel["UNRELIABLE_SLOPE"],
+}
```

Comment:
Same here.

Reply:
I changed the code to `test_dq_flags = dqflags.pixel`.